designdocs/capacity_staking_rewards_economic_model.md

```diff
@@ -0,0 +1,110 @@
+# Capacity Staking Rewards Economic Model
+
+
+## Context and Scope:
+The Frequency Transaction Payment system uses Capacity to pay for certain transactions on chain.  Accounts that wish to pay with Capacity must:
+1. Have an [MSA](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/accounts.md)
+2. Be a [Provider](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/provider_registration.md) (see also [Provider Permissions and Grants](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/provider_permissions.md))
+3. Stake a minimum amount of FRQCY token to receive [Capacity](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/capacity.md).
+
+There is also a business case for allowing any token holder to lock up its tokens in exchange for a reward - known as _staking_ - while also targeting a Provider to receive some Capacity.
+
+[//]: # (A short description of the landscape in which the new system is being built, what is actually being built.)
+[//]: # (It may also say what is not being built, and any assumptions.)
+[//]: # (Example: The proposed feature is a testing library. The context is: the library is for our chosen blockchain. The scope is: this is for a specific repository, so it's not meant to be reused. That means it won't be a separate package, and the code will be tailored for this repo. One might also say that the scope is also limited to developer testing, so it's not meant to be used in CI or a test environment such as a test blockchain network.)
+
+## Problem Statement:
+A system consisting only of providers and coinless users who delegate to providers will tend toward centralization.
```

Comment:
Maybe a quick sentence of why this leads to centralization would be nice. Maybe something like:
without a mechanism for broader participation, power and control naturally concentrate among a few providers.